openebs/elves
# Create e2e test playbook for OpenEBS support for K8s snapshot (WIP)

Performs these functions:

- Deploy operator (verify maya is running)
- Deploy application (test-pvc)
- Mount, Place data, get checksum
- Deploy openebs snapshot controller (Running)
- Get PVC details
- Create snapshot using snapshot.yaml (Running)
- List snapshot - verify successful
- List snapshot from mayactl - verify successful
- Create a clone using snap-pvc yaml
- Verify the snap-pvc is created (Running)
- Get sync status on Replica pod from "-o yaml"
- Once synched, login/mount volume
- Checksum comparison

Signed-off-by: ksatchit <karthik.s@cloudbyte.com>
